std/parallelism.d

```diff
@@ -2996,12 +2996,12 @@ public:
         }
 
     public:
-        ref T front() @property
+        ref front(this Qualified)() inout @property
         {
             return this[0];
         }
 
-        ref T back() @property
+        ref back(this Qualified)() inout @property
```

Comment:
Both `this Qualified` and `inout` is a bit much. Could we get away with just one?